pos_margin/models/pos_order.py

```diff
@@ -11,14 +11,29 @@ class PosOrder(models.Model):
 
     # Columns Section
     margin = fields.Float(
-        'Margin', compute='_compute_margin', store=True,
+        string='Margin',
+        compute='_compute_margin',
+        store=True,
         digits=dp.get_precision('Product Price'),
         help="It gives profitability by calculating the difference between"
         " the Unit Price and the cost price.")
 
+    margin_percent = fields.Float(
+        string='Margin (%)',
+        compute='_compute_margin',
+        store=True,
+        digits=dp.get_precision('Product Price'),
+    )
+
     # Compute Section
     @api.multi
-    @api.depends('lines.margin')
+    @api.depends('lines.margin', 'lines.price_subtotal')
     def _compute_margin(self):
         for order in self:
-            order.margin = sum(order.mapped('lines.margin'))
+            tmp_margin = sum(order.mapped('lines.margin'))
+            tmp_price_subtotal = sum(order.mapped('lines.price_subtotal'))
+            order.update({
+                'margin': tmp_margin,
+                'margin_percent': tmp_price_subtotal and (
+                    tmp_margin / tmp_price_subtotal * 100),
```

Comment:
```suggestion
                    tmp_margin / tmp_price_subtotal * 100) or 0.0,
```

Reply:
Hi @rousseldenis, thanks for your review.

I see your suggestion unnecessary. Don't you think ? 
``if not tmp_price_subtotal`` the result will be False, so will be converted to 0.0

But no problem to add it. Just to understand.

Reply:
Yes, indeed. I was used to that notation and be explicit :-)

Reply:
I added it.